tests/layer_tests/pytorch_tests/test_select.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+import torch
+
+
+@pytest.mark.parametrize('input_dim', list(range(4)))
+@pytest.mark.parametrize('input_index', list(range(4)))
+class TestSelect(PytorchLayerTest):
+
+    def _prepare_input(self):
+        return (np.random.randn(4, 4, 5, 5).astype(np.float32),)
+
+    def create_model(self):
+
+        input_dim = self.input_dim
```

Comment:
Please pass `input_dim` and `input_index` as parameters to `create_model` and `aten_select` init.

Reply:
I added `input_dim` and `input_index` arguments to the `create_model` method. Since I can use them as `nonlocal` variables in `aten_select` I don't think that it is required to put them explicite in the `__init__` method.

Reply:
I don't like `nonlocal` variables, if possible I'd rather avoid that.